example/package.json

```diff
@@ -14,9 +14,16 @@
   },
   "dependencies": {
     "react": "18.2.0",
-    "react-native": "0.71.4"
+    "react-native": "0.71.4",
+    "react-native-reanimated": "^3.0.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.20.0",
+    "@babel/preset-env": "^7.20.0",
+    "@babel/runtime": "^7.20.0",
+    "@react-native-async-storage/async-storage": "^1.17.11",
+    "@react-native-community/datetimepicker": "^6.7.3",
+    "@react-native-community/slider": "^4.4.2",
```

Comment:
I think we can exclude the slider and datetimepicker from the example installation

Reply:
slider and datetimepicker for storybook